cli-docker/Dockerfile

```diff
@@ -0,0 +1,28 @@
+FROM amd64/ubuntu:latest
```

Comment:
I'm gonna look into building an AArch64 binary, so we can make sure the right image for the local platform gets built.